### PR TITLE
Mercurial: Also check that the vcsRevision is not blank

### PR DESCRIPTION
**Pull request by pnowakowski**
This change is [Reviewable](https://reviewable.io/reviews/heremaps/oss-review-toolkit/100)

### CODE REVIEW

File: downloader/src/main/kotlin/vcs/Mercurial.kt [hunk 1]
```diff
@@ -71,9 +71,9 @@ object Mercurial : VersionControlSystem() {
         // We cannot detect beforehand if the Large Files extension would be required, so enable it by default.
         val extensionsList = mutableListOf(EXTENSION_LARGE_FILES)
 
-        if (vcsRevision != null) {
+        if (!vcsRevision.isNullOrEmpty()) {
```
Comment (sschuberth) [inline downloader/src/main/kotlin/vcs/Mercurial.kt]: Better extend the original code by `&& vcsRevision.isNotBlank()` to still be able to benefit from the smart cast, and to check for blank strings, too, to be on the safe side.

**Comment by pnowakowski** [comment]
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---

*[downloader/src/main/kotlin/vcs/Mercurial.kt, line 74 at r1](https://reviewable.io:443/reviews/heremaps/oss-review-toolkit/100#-L-Wap6zW-9CIvZ1dVHX-r1-74:-L-WdGxggDtLetqgFIy9:b-896fix) ([raw file](https://github.com/heremaps/oss-review-toolkit/blob/3b53344b3b7f0db9ef3624ac16077b400bbc647b/downloader/src/main/kotlin/vcs/Mercurial.kt#L74)):*
<details><summary><i>Previously, sschuberth (Sebastian Schuberth) wrote…</i></summary><blockquote>

Better extend the original code by `&& vcsRevision.isNotBlank()` to still be able to benefit from the smart cast, and to check for blank strings, too, to be on the safe side.
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/heremaps/oss-review-toolkit/100)*

**Comment by sschuberth** [comment]
Now the commit message needs to be adjusted, too :-) Maybe say

> Mercurial: Also check that the vcsRevision is not blank

**Comment by pnowakowski** [comment]
Done.

---

Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/heremaps/oss-review-toolkit/100#-:-L-WekVgmy-_Bu5mW748:b-896fix)*

**Pull request merged**